src/app-search/AppSearchDriver.js

```diff
@@ -69,6 +69,7 @@ export default class AppSearchDriver {
    * trackURLState - Boolean, track state in the url or not?
    */
   constructor({
+    endpointBase = "",
```

Comment:
I'm assuming empty string means production further down the stack?

Reply:
That's effectively what it would mean, yes.